examples/todos/src/App.tsx

```diff
@@ -1,12 +1,17 @@
-import * as React from 'react'
-import { Provider, atom, useAtom } from 'jotai'
+import { FC, FormEvent } from 'react'
+import { Provider, atom, useAtom, PrimitiveAtom } from 'jotai'
 import { Radio } from 'antd'
 import { CloseOutlined } from '@ant-design/icons'
 import { a, useTransition } from '@react-spring/web'
 
+type Todo = {
+  title: string
+  completed: boolean
+}
+
 const filterAtom = atom('all')
-const todosAtom = atom([])
-const filteredAtom = atom((get) => {
+const todosAtom = atom<PrimitiveAtom<Todo>[]>([])
+const filteredAtom = atom<PrimitiveAtom<Todo>[]>((get) => {
```

Comment:
This is not correct, because it's read-only atom. Doesn't it error?
It should work without explicit annotation.
```suggestion
const filteredAtom = atom((get) => {
```

Or, if you want to, the following:
```suggestion
const filteredAtom = atom<Atom<Todo>[]>((get) => {
```

Reply:
todosAtom type is Atom<PrimitiveAtom<Todo>[]>, so it seems filteredAtom is also Atom<PrimitiveAtom<Todo>[]> ? 🤔 
![image](https://user-images.githubusercontent.com/6883692/121764716-ebf6bb00-cb80-11eb-9ff7-b7e93769ad10.png)

Reply:
Sorry, that was my bad. I was confused with the filteredAtom type.